Kickstarter-iOS/Features/RootTabBar/RootTabBarViewController.swift

```diff
@@ -205,6 +205,8 @@ public final class RootTabBarViewController: UITabBarController, MessageBannerVi
       switch item {
       case let .home(index):
         _ = self.tabBarItem(atIndex: index) ?|> homeTabBarItemStyle
+      case let .backings(index):
+        _ = self.tabBarItem(atIndex: index) ?|> backingsTabBarItemStyle
```

Comment:
Is it worth moving away from prelude operators like this as we come across them?